databuilder/extractor/ver

```diff
@@ -0,0 +1,139 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+from collections import namedtuple
+
+from pyhocon import ConfigFactory, ConfigTree
```

Comment:
could you make it a .py file

Reply:
Oops, my bad! Sorry about that. Not sure how it got added under that name. Anyway, I have fixed it and also made sure that the tests pass.